lib/ingredients/NgAnnotate.js

```diff
@@ -0,0 +1,35 @@
+const Ingredient = require('./Ingredient');
+const ngAnnotate = require('ng-annotate');
+const fs = require('fs');
+
+class NgAnnotate extends Ingredient {
+    constructor(config, options) {
+        super(config, options);
+
+        this.ingredientConfig = Object.assign({
+            single_quotes: true,
+            remove: true,
+            add: true
+        }, this.getIngredientConfig('NgAnnotate'));
+    }
+
+    mergeJsContent(files, templateCache = '') {
+        return files.map(function(file) {
```

Comment:
Yeah I was thinking the same when I wrote it actually, but I'm not sure if it will actually speed things up unless we let Node run over different cores. Which might actually be pretty awesome as well.